example/src/utils.ts

```diff
@@ -4,7 +4,7 @@ export function sleep(ms: number): Promise<void> {
   })
 }
 
-export const PLUGGY_BANK_CONNECTOR = 0
+export const PLUGGY_BANK_CONNECTOR = 2
```

Comment:
Maybe we could expand the variable name a bit? To make it clearer which type of pluggy bank conector we are referencing